be/src/storage/rowset/ordinal_page_index.cpp

```diff
@@ -130,8 +132,8 @@ Status OrdinalIndexReader::_do_load(const IndexReadOptions& opts, const OrdinalI
     RETURN_IF_ERROR(reader.parse(body, footer.index_page_footer()));
 
     _num_pages = reader.count();
-    _ordinals.resize(_num_pages + 1);
-    _pages.resize(_num_pages);
+    _ordinals = std::make_unique<ordinal_t[]>(_num_pages + 1);
```

Comment:
This will cause more small memory allocation. If all the size ordinal index size is the same, memory can be reused. So how about to record the capacity to reduce memory allocation

Reply:
The number of pages in each Column is different, Do you mean that the memory of OrdinalIndex is allocated by a unified fixed Size memory allocator？

Reply:
The memory fragmentation rate of the current implementation should be the same as the previous implementation, right?